src/vs/workbench/contrib/chat/browser/actions/chatContextActions.ts

```diff
@@ -50,7 +53,15 @@ export interface IFileQuickPickItem extends IQuickPickItem {
 	name: string;
 	value: URI;
 	isDynamic: true;
+	resource: URI;
+}
 
+export interface ImageQuickPickItem extends IQuickPickItem {
+	kind: 'image';
+	id: string;
+	name: 'image';
```

Comment:
Why is `name` hardcoded? I see a check below but shouldn't that be checking `kind` instead?